### PR TITLE
WIP - fix: card reputation

### PR DESCRIPTION
**Pull request by CzarekDryl**
https://tw.auroracreation.com/app/tasks/15496505

### CODE REVIEW

File: src/components/v5/common/TeamReputationSummary/TeamReputationSummary.tsx [hunk 1]
```diff
@@ -8,61 +8,100 @@ import { TextButton } from '~v5/shared/Button';
...
           <ul>
-            {teams?.map((team, index) => {
+            {sortedTeams?.map((team, index) => {
               const { nativeId } = team;
               return (
                 index < 5 && (
-                  <li key={nativeId} className="flex items-center text-sm mb-3">
+                  <li
+                    key={nativeId}
+                    className="flex items-center text-sm mb-3 last:mb-0"
```
Comment (krawczykkatarzyna) [inline src/components/v5/common/TeamReputationSummary/TeamReputationSummary.tsx:95]: maybe `gap` will be better then `margin`?

File: src/components/v5/common/TeamReputationSummary/TeamReputationSummary.tsx [hunk 2]
```diff
@@ -8,61 +8,100 @@ import { TextButton } from '~v5/shared/Button';
...
+            (acc, { reputation: teamReputation }) =>
+              acc.plus(teamReputation || 0),
+            new Decimal(0),
+          ) || 0,
+        )
+        .abs()
+        .toString(),
+    }))[0];
+
+  const showOthers = sortedTeams && sortedTeams?.length > 5;
```
Comment (krawczykkatarzyna) [inline src/components/v5/common/TeamReputationSummary/TeamReputationSummary.tsx:54]: please assign  the `5` to value in consts file.

File: src/components/v5/common/TeamReputationSummary/TeamReputationSummary.tsx [hunk 3]
```diff
@@ -8,61 +8,100 @@ import { TextButton } from '~v5/shared/Button';
...
 import { formatText } from '~utils/intl';
+import { useActionSidebarContext } from '~context/ActionSidebarContext';
+import { Actions } from '~constants/actions';
 
 const displayName = 'v5.common.TeamReputationSummary';
 
 const TeamReputationSummary: FC = () => {
   const { colony } = useColonyContext();
-  const { nativeToken, domains, reputation } = colony || {};
+  const { domains, reputation } = colony || {};
```
Comment (krawczykkatarzyna) [inline src/components/v5/common/TeamReputationSummary/TeamReputationSummary.tsx:21]: a lot of logic is in that file, maybe will be better to move it to hooks file?

**Comment by chmanie** [comment]
Hey, what's the status of this? Still WIP?

**Comment by arrenv** [comment]
Closing as no longer needed.

**Pull request closed without merging**